environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+
+## Quickstart
+
+### 1. Clone the kcp-edge repo:
+
+```shell
+git clone https://github.com/kcp-dev/edge-mc.git  kcp-edge
+```
+
+### 2. Build the kcp-edge binaries:
+
+```shell
+cd kcp-edge
+make build
+```
+
+The kcp-edge binaries is created inside the `/bin` directory. Add this directory to your $PATH:
+
+```shell
+export PATH="$PATH:$(pwd)/bin"
+```
+
+
+### 3. Create your own edge infrastructure (edge clusters) - Bring Your Own Cluster (BYOC)
+
+Create your edge cluster or bring your own k8s edge cluster. In this example, we use [kind](https://kind.sigs.k8s.io/) to create an edge cluster that we name “florin”:
+
+```shell
+kind create cluster --name florin
+```  
+
+### 4. Deploy the kcp-edge platform:
```

Comment:
Steps a and b are kcp rather than kcp-edge (edge-mc).  See my earlier discussion with @francostellari about keeping clear on the fact that edge-mc is embedded in a context --- currently kcp but something more general in the future --- that provides a runtime-variable fleet of logical clusters.  That larger context is not part of edge-mc.

Reply:
fixed